web/css/text-area.css

```diff
@@ -0,0 +1,85 @@
+.noora-text-area {
+  display: flex;
+  flex-direction: column;
+  align-self: stretch;
+  gap: var(--noora-spacing-3);
+
+  & > [data-part="wrapper"] {
+    padding: var(--noora-spacing-5);
+    box-shadow: var(--noora-border-light-default);
+    border-radius: var(--noora-radius-medium);
+    background: var(--noora-surface-background-primary);
+    display: flex;
+    position: relative;
+  }
+
+  &[disabled] > [data-part="wrapper"] {
+    background: var(--noora-surface-background-secondary);
+  }
+
+  &:not([disabled]) {
+    & > [data-part="wrapper"] {
+      cursor: text;
+
+      &:hover,
+      &:focus {
+        background: var(--noora-surface-background-secondary);
+        box-shadow: var(--noora-light-border-focus);
+      }
+    }
+  }
+
+  &[data-error] > [data-part="wrapper"] {
+    box-shadow: var(--noora-border-light-error);
+
+    &:hover {
+      box-shadow: var(--noora-border-light-error-focus);
+    }
+  }
+
+  & textarea {
+    flex-grow: 1;
+    outline: none;
+    border: none;
+    background: none;
+    color: var(--noora-surface-label-primary);
+    font: var(--noora-font-weight-regular) var(--noora-font-body-medium);
+    &::-webkit-resizer {
```

Comment:
the resizer is only customizable in Safari and Chrome. The default resizer is shown in Firefox. The alternative would be to reimplement the resizer, which we'd like to avoid.